src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -12,6 +12,7 @@ import { IConfigurationService } from 'vs/platform/configuration/common/configur
 import { dispose, Disposable, DisposableStore } from 'vs/base/common/lifecycle';
 import { Registry } from 'vs/platform/registry/common/platform';
 import { coalesce } from 'vs/base/common/arrays';
+import { IResourceEditorInputType } from 'vs/workbench/services/editor/common/editorService';
```

Comment:
Not a big fan of this dependency, can be cleaned up if we move `IResourceEditorInputType` into e.g. `workbench/common/editor`, maybe even rename it to `IUntypedEditorInput`?